extensions/amp-sidebar/0.1/amp-sidebar.css

```
@@ -80,6 +80,10 @@ amp-sidebar[side], .i-amphtml-sidebar-mask {
   margin: 0;
 }
 
+.amphtml-sidebar-mask {
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
 .i-amphtml-sidebar-mask {
```

If we're exposing a public classname I don't think we need the private one.

We also usually prefix with `amp-` for public classnames, not `amphtml-`

```suggestion
.amp-sidebar-mask {
  background-color: rgba(0, 0, 0, 0.5);
```

---

Switching to `amp` instead of `amphtml`.

---

EDIT: We're using the private className in other places, fine to leave as is.

---

Keeping both names to prevent a larger scope.